### ISSUE TITLE
chapter 2 and 3 some examples not working

### ISSUE THREAD

**Issue by kdhl** [openlayersbook/openlayersbook#4]
The bindTo example 2.3 and the Target practice example of 3.3 do not show any visible reaction to the selection/button press (I can see the DIV being changed in the inspector). Tried this in chrome and internet explorer on windows7 and chromium and firefox on lubuntu? I'm using 4.1.0.

Any idea what's going on? Differences in OL3.x vs OL4.x ?

**Comment by ThomasG77** [comment]
Yes, in fact, difference between OpenLayers 3.0 and more recent 3.5.
See https://github.com/openlayers/openlayers/releases/tag/v3.5.0 for instructions to fixing. I plan to update the content but too busy at the moment (there is already more than 2 years since we release the book associated with samples)
